scanners/android/android.go

```diff
@@ -129,6 +137,15 @@ func parseProjects(searchDir string, projectRoots []string) ([]Project, error) {
 	return projects, nil
 }
 
+func usesKotlinBuildScripts(projectRoot string) bool {
+	return fileExists(filepath.Join(projectRoot, gradleKotlinBuildFile)) && fileExists(filepath.Join(projectRoot, gradleKotlinSettingsFile))
+}
+
+func fileExists(path string) bool {
```

Comment:
NOTE: in the spmproject.go we also have a `fileExists`, shouldn't we share these codes?